src/ducks/balance/History.jsx

```diff
@@ -1,32 +1,87 @@
 import React, { Component } from 'react'
 import PropTypes from 'prop-types'
 import cx from 'classnames'
+import * as d3 from 'd3'
 import { translate } from 'cozy-ui/react'
 import { Figure } from 'components/Figure'
+import { groupBy, sumBy, uniq } from 'lodash'
+import { format as formatDate, parse as parseDate } from 'date-fns'
+import LineChart from 'components/Chart/LineChart'
+import { getBalanceHistories } from './helpers'
 import styles from './History.styl'
 
 class History extends Component {
+  getCurrentBalance() {
+    return sumBy(this.props.accounts, a => a.balance)
+  }
+
+  getBalanceHistory() {
+    const { accounts, transactions } = this.props
+    const balanceHistories = getBalanceHistories(accounts, transactions)
+    const balanceHistory = Object.entries(balanceHistories.all)
```

Comment:
```
import { sortBy } from 'lodash
const balanceHistory = sortBy(Object.entries(balanceHistories.all), x => x[0])
```

Reply:
I don't understand what is this first element, is it the date ? Maybe this sort logic could be extracted into a function

```
const sortBalanceHistoryByDate = sortBy(balanceHistories, x => x[0])
```

Reply:
:heavy_check_mark: